docs/src/Fieldset.doc.js

```diff
@@ -0,0 +1,222 @@
+// @flow strict
+import type { Node } from 'react';
+import { Fieldset } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Fieldset"
+    description="Creates an accessible fieldset and legend for a group of related form items"
+    defaultCode={`
+      function RadioButtonExample() {
+        const [favorite, setFavorite] = React.useState(undefined);
+
+        return (
+          <Fieldset legend="What is your favorite pet?">
+            <Flex direction="column" gap={2}>
+              <RadioButton
+                checked={favorite === 'dogs'}
+                id="favoriteDog"
+                label="Dogs"
+                name="favorite"
+                onChange={() => setFavorite( 'dogs' )}
+                value="dogs"
+              />
+              <RadioButton
+                checked={favorite === 'cats'}
+                id="favoriteCat"
+                label="Cats"
+                name="favorite"
+                onChange={() => setFavorite( 'cats' )}
+                value="cats"
+              />
+              <RadioButton
+                checked={favorite === 'plants'}
+                id="favoritePlants"
+                label="Plants"
+                name="favorite"
+                onChange={() => setFavorite( 'plants' )}
+                value="plants"
+              />
+            </Flex>
+          </Fieldset>
+
+        );
+      }
+`}
+  />,
+);
+
+card(
+  <PropTable
+    Component={Fieldset}
+    props={[
+      {
+        name: 'legend',
+        type: 'string',
+        required: true,
+        description:
+          "Content for this fieldset's legend that clearly and concisely describes the question being asked.",
+      },
+      {
+        name: 'legendDisplay',
+        type: "'visible' | 'hidden'",
+        defaultValue: 'visible',
+        description:
+          'Whether the legend should be visible or not. If `hidden`, the legend is still available for screen reader users, but does not appear visually.',
+      },
```

Comment:
I'd add .. See (legent visibility)[] to learn more.

Reply:
Maybe OOS, but we could repro our settings page 
![image](https://user-images.githubusercontent.com/10593890/117051354-70cdf980-ace4-11eb-8eb0-dfd13fe37a23.png)
very simplified, but to show a more realistic usage and different fieldsets working together withng a form